cmd/premain-graphene/main.go

```diff
@@ -40,11 +39,10 @@ func main() {
 	argv := toCArray(os.Args)
 	envp := toCArray(os.Environ())
 
-	// spawn service
-	if res := C.posix_spawn(nil, C.CString(service), nil, nil, &argv[0], &envp[0]); res != 0 {
+	// launch service
+	if res := C.execve(C.CString(service), &argv[0], &envp[0]); res != 0 {
```

Comment:
Can you try if https://golang.org/pkg/syscall/#Exec works here?

Reply:
Works, so no more CGo needed here. Great!